models/shopify_holistic_reporting__orders_attribution.sql

```diff
@@ -52,6 +52,7 @@ with orders as (
         events.event_id as last_touch_event_id,
         events.occurred_at as last_touch_event_occurred_at,
         events.type as last_touch_event_type,
+        events.integration_id as last_touch_integration_id,
         events.integration_name as last_touch_integration_name,
         events.integration_category as last_touch_integration_category,
```

Comment:
While the integration fields were affected upstream in Klaviyo, now coming a different source (metric) since the initial one (integration) was deprecated, the downstream isn't impacted as seen here. However we did introduce a new field `integration_id`, so I added that here